targets/XPlatCppSdk/source/code/include/playfab/PlayFabError.h.ejs

```diff
@@ -40,4 +40,5 @@ namespace PlayFab
     };
 
     typedef std::function<void(const PlayFabError& error, void* customData)> ErrorCallback;
+    typedef std::function<void(const std::exception& exception)> ExceptionCallback;
```

Comment:
>& [](start = 51, length = 1)

Please pass the copy of exception here, not a reference. We had a long discussion with Deb about this (i initially also proposed to use a reference). The main reason is we don't want external user to accidentally use it AFTER the exception object was destroyed (e.g. they save it somewhere and then attempt to dereference later). Talk to Deb if you have a different opinion.